### DISCUSSION TITLE
Github account recovery

### DISCUSSION THREAD

**Discussion by Rajeshkannan-Muthukumar** [community/community#59935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi , this is Rajeshkannan-Muthukumar. Few days ago ,my github acoount got hacked, the hacker did something. After that i reseted my Password, then my github profile is appearing only for me, when i'm logged in. It is not available for public, how to make it appear on public search... [www.github.com/Rajeshkannan-Muthukumar](http://www.github.com/Rajeshkannan-Muthukumar)

**Comment by cehfisher** [comment]
Hi @Rajeshkannan-Muthukumar - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
